FEM/BHE_1U.h

```diff
@@ -0,0 +1,339 @@
+/**
+* \file BHE_1U.h
+* 2014/06/04 HS inital implementation
+* class of borehole heat exchanger with 1 U-tube
+*
+*/
+
+#ifndef BHE_1U_H
+#define BHE_1U_H
+
+#include "BHEAbstract.h"
+#include "makros.h"
+#include "tools.h"
+
+namespace BHE  // namespace of borehole heat exchanger
+{
+
+	class BHE_1U : public BHEAbstract
+	{
+	public:
+		/**
+		* constructor
+		*/
+        BHE_1U(const std::string name             /* name of the BHE */,
+               BHE::BHE_BOUNDARY_TYPE bound_type  /* type of BHE boundary */,
+               bool   if_use_ext_Ra_Rb            /* whether Ra and Rb values are used */,
+			   bool user_defined_R_vals           /* when user defined R values are used*/,
+               double my_L = 100                  /* length/depth of the BHE */,
```

Comment:
indentation. There are several class have the almost the same arguments for their constructors. It might better   to introduce a struct for these data to avoid code duplication.